examples/website-demos/social-messaging/src/components/ChatContainer/ChatContainer.tsx

```diff
@@ -84,23 +86,63 @@ export const ChatContainer: React.FC = () => {
         };
     }, []); // eslint-disable-line
 
+    const customRenderFilter = (channels: TypeChannel[]) => {
+        const getTotalChatUnreadCount =
+            channels?.map(channel => channel.countUnread()).reduce((total, count) => total + count, 0);
```

Comment:
looks like the function param isn't optional so it will always be truthy and this map will run regardless, even if empty array